language/stdlib/modules/AccountLimits.move

```diff
@@ -185,63 +186,74 @@ module AccountLimits {
     // specified the `limits_definition` passed in.
     fun can_receive<CoinType>(
         amount: u64,
-        receiving: &mut Window,
+        receiving: &mut Window<CoinType>,
     ): bool acquires LimitsDefinition {
-        let limits_definition = borrow_global_mut<LimitsDefinition>(receiving.limits_definition);
-        // If the limits ares unrestricted then no more work needs to be done
-        if (is_unrestricted(limits_definition)) return true;
+        let limits_definition = borrow_global_mut<LimitsDefinition<CoinType>>(receiving.limit_address);
+        // If the limits ares unrestricted then only track the balance
```

Comment:
```suggestion
        // If the limits are unrestricted then only track the balance
```

Reply:
Could also be useful to explain why we need to track the balance in this case (is it because balance holding fees are tracked + assessed independently of account limits)?

Reply:
Will do. It's more so that we can track the balance of the entire entity. If we didn't track this from the start, there would be no easy way to impose a balance limit later on since we wouldn't know (on chain) how much is held across all of the entity's accounts.